moqu/core/src/main/java/io/quarkiverse/openapi/moqu/model/Header.java

```diff
@@ -0,0 +1,12 @@
+package io.quarkiverse.openapi.moqu.model;
+
+import java.util.Set;
+
+/**
+ * Represents an HTTP header with a name and a set of associated values.
+ *
+ * @param name the name of the HTTP header (e.g., "Accept", "Content-Type").
+ * @param value the set of values associated with the header, allowing multiple values (e.g., "application/json", "text/html").
+ */
+public record Header(String name, Set<String> value) {
```

Comment:
```suggestion
public record Header(String name, List<String> value) {
```
Although duplicating string values for a header seems weird, it can actually make sense in some scenarios. Also, header values are an ordered collection, List should be used rather than Set